packages/utils/src/lib/crypto.ts

```diff
@@ -38,36 +53,60 @@ export const formatNewmAmount = (
  */
 export const formatAdaAmount = (
   amount?: number,
-  options: FormatCurrencyOptions = { includeSymbol: true, precision: 2 }
+  options: FormatCurrencyOptions = {
+    includeCurrencySymbol: true,
+    includeEstimateSymbol: false,
+    precision: 2,
+  }
 ) => {
   if (!amount) return "₳ 0";
 
-  const { includeSymbol = true, precision = 2 } = options;
+  const {
+    includeCurrencySymbol = true,
+    includeEstimateSymbol = false,
+    precision = 2,
```

Comment:
No changes to ADA besides adding the estimated symbol. I don't think we currently have any locations that would need to list "less than .." for ADA values. Precision might need to be evaluated at that point as well. Will bring it up to product to have an eye out.